hack/app-sre/kustomization.yaml

```diff
@@ -0,0 +1,46 @@
+# This kustomize configuration deploys what is required for the hive-operator
+# *without* OLM.
+# Except:
+# It does not contain a HiveConfig.
+
+# Adds namespace to all resources.
+namespace: hive
+
+# The CRD resources are synced automatically by the `build-app-sre-template` make target.
+# The other resources should be curated manually.
+resources:
+- ../../config/namespace.yaml
```

Comment:
the namespace is created by app-interface and shouldn't be applied.

Reply:
Ack, removed.